dependabot/util.py

```diff
@@ -0,0 +1,16 @@
+import github3.repos
+import typing
+
+import ccc.github
+
+
+def repositories_for_org(
+    full_org_name: str,  # e.g. github.com/gardener
```

Comment:
not sure I like `full_org_name` :-/
how about qualified_org, or github_org_url, or hostname_and_org?

Reply:
`github_org_url` sounds good